code/game/machinery/doors/door.dm

```diff
@@ -492,21 +498,65 @@
 		sleep(0.1 SECONDS)
 		open()
 
+/obj/machinery/door/proc/hurt_human(var/mob/living/carbon/victim)
+	if (prob(CRUSH_BEHEAD_CHANCE))
```

Comment:
also for future reference its probably better to do a good 'ol

```
	if (prob(BASE_DELIMB_CHANCE))
	    if (head)
                  if (prob(BEHEAD_CHANCE))
```

so delimb chance isnt artificially reduced by someone having no head